### PR TITLE
Day 13 added

### PR DESCRIPTION
**Pull request by TsegtsErdene**

### CODE REVIEW

**Comment by lkhagvaa2444** [comment]
dockerfile bolon dockercompose shalgav.
dockerfile ajilllaj bn.
dockercompose ajilluulav. ajillaj bn. gehdee http://localhost:3001/api/#/ buyu api ni ajillahgui bn.

sain tal: gol ym ni ajillaj bn
sul tal: api panel ajillahgui bn.
Screenshot (130)
Screenshot (131)
user-images.githubusercontent.com/77320148/223349242-69aa098c-54d2-4d94-8496-ec41e6d0a893.png)
Screenshot (132)

File: week1/tsegts-erdene/Day 11/portfolio/package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "homepage": "https://Tsegts-erdene.github.io/home",
```
Comment (biligunb) [inline week1/tsegts-erdene/Day 11/portfolio/package.json:2]: @TsegtsErdene github.io link нь болохгүй байх шиг байхын?

File: week1/tsegts-erdene/Day 11/portfoliolink [hunk 2]
```diff
@@ -0,0 +1 @@
+https://tsegtserdene.github.io/portfolio/
```
Comment (biligunb) [inline week1/tsegts-erdene/Day 11/portfoliolink:1]: @TsegtsErdene aan ene bsimu?
 ygd iim oor URL deer bgan?

**Review by biligunb — APPROVED** [review]
MERGE after comments

**Pull request merged**